glean_parser/metrics.py

```diff
@@ -306,6 +306,13 @@ class Labeled(Metric):
     labels: List[str] = None
     labeled = True
```

Comment:
One thing that might be worrying: during development I accidentally left this value as `False` and all the testsstill passed... Maybe there should some more specific tests that would prevent that from being the case.

Reply:
Yeah -- that's a good idea.